src/style/variables.scss

```diff
@@ -39,8 +39,8 @@ $mdc-theme-text-secondary-on-background: rgba(0, 0, 0, 0.54);
 @import "@limetech/mdc-theme/mdc-theme";
 
 $mdc-typography-styles-headline1: (
-    font-size: pxToRem(16),
-    line-height: pxToRem(14)
+    font-size: pxToRem(22),
+    line-height: pxToRem(22)
```

Comment:
At first when I was chaning the size in the header of the collapsible section I thought it was kind of strange that material had pretty much the same sizes for all the headlines, and they looked quite similar.

Now I found these :smile: I find it a bit strange though that we have set all of them to pretty much the same size as they start to loose their meaning if they are all the same. I guess we should go through these at some point and give them a proper size so we can use them

Reply:
Oh, and I took the liberty of changing one of them now that we did not seem to use anywhere apart from in the new form component

Reply:
Haha! Yeah, I have no idea why they are all the same 😂 I guess maybe it was done in some effort to have the new components from lime-elements fit in with the old stuff from the webclient back in the day… 🤔